sites/svelte.dev/content/blog/2023-05-01-whats-new-in-svelte-may-2023.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "What's new in Svelte: May 2023"
+description: "TBD"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+TBD... Svelte Hackathon winner goes here too.
```

Comment:
These won't be annunced until May 6th and it'll be a segment on svelte summit. maybe put a promotion for that here instead? cc @kevmodrome

Reply:
perfect. I'll make this a mention of the forthcoming announcement, then :)